polycommit/src/lib.rs

```diff
@@ -711,7 +724,7 @@ pub mod tests {
                 let point = F::rand(rng);
                 for (i, label) in labels.iter().enumerate() {
                     query_set.insert((label.clone(), (format!("rand_{}", point_id), point)));
-                    let value = polynomials[i].evaluate(point);
+                    let value = polynomials[i].evaluate(&point);
```

Comment:
in order to avoid indexing, `polynominals` could be `.zip()`ped with `labels` and iterated together